python/taichi/__init__.py

```diff
@@ -29,6 +29,7 @@
     'kernel_profiler_total_time': 'profiler.get_kernel_profiler_total_time',
     'set_kernel_profiler_toolkit': 'profiler.set_kernel_profiler_toolkit',
     'set_kernel_profile_metrics': 'profiler.set_kernel_profiler_metrics',
+    'print_memory_profile_info': 'profiler.print_memory_profile_info',
```

Comment:
```suggestion
    'print_memory_profile_info': 'profiler.print_memory_profiler_info',
```

Reply:
BTW this line may be moved to L24 to group things better (scope profiler, memory profiler, kernel profiler).